dockertest/subtest.py

```diff
@@ -24,6 +24,11 @@
 import version
 import config
 import subtestbase
+# There is a bug in Pylint + virtualenv that makes this fail in Travis CI
+# https://github.com/PyCQA/pylint/issues/73
+from distutils.version import LooseVersion  # pylint: disable=E0611,F0401
+from dockercmd import DockerCmd
+from output import DockerVersion
```

Comment:
Okay, this is probably fine.  I don't think we'll ever need to ``import subtest`` inside of ``output``, no?